Makefile

```
@@ -36,6 +38,11 @@ ifeq ($(UNAME), Darwin)
 deps: deps-macos
 endif
 
+install_nextest:
```

Just to be consistent with the rest of the target names:

```suggestion
install-nextest:
```